tests/acceptance/features/apiSpaces/resharing.feature

```diff
@@ -0,0 +1,85 @@
+    #And user "Alice" <AliceVisible> included in the response $TODO: How to check alice is owner every time?
+    And the response should contain <numVisibleShares> entries
+    And user "Brian" <BrianVisible> included in the response
+    And user "Carol" <CarolVisible> included in the response
+    And user "Damian" <DamianVisible> included in the response
+    Examples:
+      | user   | numVisibleShares | BrianVisible  | CarolVisible   | DamianVisible  |
+      | Brian  | 2                | should be     | should be      | should not be  |
+      | Carol  | 2                | should not be | should be      | should be      |
+      | Damian | 1                | should not be | should not be  | should be      |
+
+  Scenario: Owners can see all the chain:
+    When user "Alice" gets all the shares inside the folder "folder" using the sharing API
+    Then the OCS status code should be "100"
+    And the HTTP status code should be "200"
+    And the response should contain 3 entries
+    And user "Brian" should be included in the response
+    And user "Carol" should be included in the response
+    And user "Damian" should be included in the response
+
+  Scenario: You can't share with more permissions than you have
+    When user "Ember" has been created with default attributes and without skeleton files    
+    And user "Damian" shares folder "Shares/folder" with user "Ember" with permissions "31" using the sharing API
+    Then the OCS status code should be "404"
+    And the OCS status message should be "Cannot set the requested share permissions"
+
+  Scenario Outline: Editing reshares
+    When user "Fred" has been created with default attributes and without skeleton files    
+    And user "Carol" has shared folder "Shares/folder" with user "Fred" with permissions "17"
+    And user "Fred" accepts share "/folder" offered by user "Carol" using the sharing API
+    And as "Fred" folder "Shares/folder" should exist
+    Then user "<user>" updates the last share using the sharing API with
+      | permissions | 31 |
+    And the OCS status code should be "<code>"
+    And user "Fred" <canUpload> able to upload file "filesForUpload/textfile.txt" to "/Shares/folder/textfile.txt"
+    Examples:
+      | user  | code | canUpload     |
+      | Alice | 100  | should be     |
+      | Brian | 998  | should not be |
+      | Carol | 100  | should be     |
+
+  Scenario Outline: Deleting reshares
+    When user "Gina" has been created with default attributes and without skeleton files    
+    And user "Carol" has shared folder "Shares/folder" with user "Gina" with permissions "17"
+    And user "Gina" accepts share "/folder" offered by user "Carol" using the sharing API
+    And as "Gina" folder "Shares/folder" should exist
+    Then user "<user>" deletes the last share using the sharing API
+    And the OCS status code should be "<code>"
+    And as "Gina" folder "Shares/folder" <exists>
+    And as "Carol" folder "Shares/folder" should exist
```

Comment:
@phil-davis this test is failing since the last master rebase. I think it is because of the line
```
Then user "<user>" deletes the last share using the sharing API
```
and the recently merged "last share" improvements.
I believe my testcode is not correct any more (or it never was).
What I am trying to do is: As user "<user>" delete the share to user "Gina"

Can you help me how I can achieve this most easily?

Reply:
Sorry I took so long! Yes, there was a regression in the core test code for this type of scenario. I could reproduce it in core by creating a similar scenario there locally and running it. PR https://github.com/owncloud/core/pull/40133 should fix it. I will get the core commit id updated tomorrow morning...

Reply:
@kobergj the core commit id has been bumped in PR https://github.com/owncloud/ocis/pull/3961

Please rebase and these tests should work better.